gulpfile.js

```diff
@@ -0,0 +1,75 @@
+const { src, dest } = require('gulp');
```

Comment:
I'm not sure why we need Gulp for this ;-/ What exactly this code is doing?

Reply:
Let’s not start this discussion. You’re free to do this without gulp after the workshop :)

Reply:
The code is an adaptation of splitTask by Kent that I tried before but without the I/O code (90%).